api/handler/multipart_upload.go

```diff
@@ -436,18 +436,9 @@ func (h *handler) CompleteMultipartUploadHandler(w http.ResponseWriter, r *http.
 		h.log.Error("couldn't send notification: %w", zap.Error(err))
 	}
 
-	bktSettings, err := h.obj.GetBucketSettings(r.Context(), bktInfo)
-	if err != nil {
-		h.logAndSendError(w, "could not get bucket settings", reqInfo, err)
-	}
-
-	p := &layer.DeleteObjectParams{
-		BktInfo:     bktInfo,
-		BktSettings: bktSettings,
-		Objects:     []*layer.VersionedObject{{Name: initPart.Name}},
-	}
-	if _, err = h.obj.DeleteObjects(r.Context(), p); err != nil {
-		h.logAndSendError(w, "could not delete init file of multipart upload", reqInfo, err, additional...)
+	if err = h.obj.AbortMultipartUpload(r.Context(), uploadInfo); err != nil {
```

Comment:
It's a little confused to see `AbortMultipartUpload` after `CompleteMultipartUpload`. In addition it leads to overhead to get all parts again. I think we can just use `DeleteSystemObject` to delete `initPart` here.